exercises/concept/chaitanas-colossal-coaster/list_methods_test.py

```diff
@@ -88,9 +88,9 @@ def test_add_me_with_my_friends_end_of_queue(self):
         params = (["Natasha", "Steve", "Tchalla", "Wanda", "Rocket"], 5, "Bucky")
         result = ["Natasha", "Steve", "Tchalla", "Wanda", "Rocket", "Bucky"]
 
-        self.assertListEqual( add_me_with_my_friends(*params), result,
-            msg="The person was added to the wrong location in the queue or was not added at all"
-        )
+        self.assertListEqual(add_me_with_my_friends(*params), result,
+                             msg="The person was added to the wrong location in the queue or was not added at all"
```

Comment:
Indentation seems to be less than expected here, based on other areas that are indented differently.

Reply:
```suggestion
              msg="The person was added to the wrong location in the queue or was not added at all"
```
This passed linting without complaint, but I will back this one out to be consistent within this file.